src/apps/bridge/bridgePowerController.cpp

```diff
@@ -362,7 +362,12 @@ uint32_t BridgePowerController::getCurrentTimeS() {
 uint32_t BridgePowerController::_alignNextInterval(uint32_t nowEpochS,
                                                    uint32_t lastIntervalStartS,
                                                    uint32_t sampleIntervalS) {
-  uint32_t alignedEpoch = lastIntervalStartS + sampleIntervalS;
+  uint32_t alignedEpoch = 0;
```

Comment:
Minor thing but could you remove the alignedEpoch = 0 line and add the uint32_t at the beginning of line 370 below. Since alignedEpoch is only needed on 371 and not within the if clause, that would make this PR is only the addition of the check for lastIntervalStartS == 0.